eisbuk-admin/src/components/atoms/AttendanceCard/__tests__/AttendanceCard.test.tsx

```diff
@@ -24,5 +29,9 @@ describe("AttendanceCard", () => {
       screen.getByText("Walter");
       screen.getByText(Category.Competitive);
     });
+    test("should call markAttendance with correct args", () => {
+      userEvent.click(screen.getByText("Mark Walter as present"));
```

Comment:
No need for `userEvent` here, to keep things as minimal as possible, use the `screen.getByText("Mark Walter as present").click()`. Also, what happened to the :+1: :-1: ?